p0_user_stories.md

```diff
@@ -31,22 +31,22 @@ other pods should be blocked from accessing it via any IP.
 
 - I want all namespaces matching X to be completely 100% locked-down by default; no pods can talk to any other pods until the developers specify policies allowing it - I may also want the converse situation for namespace Y.
 
-- I want to target all apps with network policies, but most of my apps need apiserver access, and i dont know the k8s service IP because all my clusters have different Service CIDRs.
+- I want to target all apps with network policies, but most of my apps need apiserver access, and i dont know the k8s service IP because all of my clusters have different Service CIDRs.
 
-- I want to block all pods from being able to reach ports on nodes. And the rule should automatically cover all nodes, and should cover all IPs on those nodes, except for port 53.
+- I want to block all pods from being able to reach ports on nodes. The rule should automatically cover all nodes, and all IPs on those nodes, except for port 53.
```

Comment:
We currently do not do anything with pods talking to things on the actual node right?